quadlet/ollama/ollama.container

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=An ollama container
+After=network-online.target
+
+[Container]
+Image=docker.io/ollama/ollama:latest
+AutoUpdate=registry
+ContainerName=ollama
+Network=ollama.network
+Volume=ollama.volume:/root/.ollama
+PublishPort=11434:11434
```

Comment:
Since you expect the users to interact with the WebUI and both containers run in the same network, you don't need to publish the Ollama ports. Maybe just comment out this line and add a comment about publishing it in case the user wants to expose the Ollama an endpoint